portality/templates/includes/footer.html

```diff
@@ -15,8 +15,15 @@ <h2 class="sr-only">
                 Legal &amp; admin
             </h2>
             <div>
+                <a href="/" class="logo">
+                    <h2 class="sr-only">The Directory of Open Access Journals</h2>
+                    {% include "includes/svg/logo.svg" %}
+                    <span class="logo__tagline">
+                      Open. <br/>Global. <br/>Trusted.
+                    </span>
+                </a>
                 <p>
-                    © DOAJ 2020 default by all rights reserved <a href="{{ url_for('doaj.terms') }}" style="text-decoration: underline;">unless otherwise specified</a>.
+                    © DOAJ 2022 default by all rights reserved <a href="{{ url_for('doaj.terms') }}" style="text-decoration: underline;">unless otherwise specified</a>.
```

Comment:
@richard-jones This isn’t ideal of course, but couldn’t figure out how to get the current year in Jinja templates

Reply:
Apparently we used to have this and we must have just lost it during the redesign.  If you just do {{ current_year }} it'll give you the current year

Reply:
Thank you! It works — committed.